apps/laboratory/tests/shared/utils/logs.ts

```diff
@@ -0,0 +1,17 @@
+/* eslint no-console: 0 */
+
+const TIMING_LOGS_ENABLED = process.env['TIMING_LOGS'] === 'true' || false
+
+export function timeStart(label?: string | undefined) {
```

Comment:
Why do you allow undefined? I feel like this should just be `label: string`

Reply:
Following the definition of `console.time`, you can actually just call it like that without any param